### PR TITLE
[gala]Intercept runtime errors while calling into GDB pretty printer 

### PR DESCRIPTION
**Pull request by oontvoo**
... and display them as a "fake children" for readability.

### CODE REVIEW

**Comment by oontvoo** [comment]
@slackito

File: gdb/printing.py [hunk 1]
```diff
@@ -249,20 +248,28 @@ def _get_children(self, max_count: int) -> None:
                 # In order to do this we need our error message as a valid C++
                 # expression. We pack it into a raw string literal with a custom
```
Comment (slackito) [inline gdb/printing.py]: Please move this comment to the line where `CreateValueFromExpression` is called with the `R"GALA_ERROR(...)GALA_ERROR"` string literal.

Comment (oontvoo) [reply gdb/printing.py]: done - moved the msg

File: gdb/printing.py [hunk 2]
```diff
@@ -246,23 +245,34 @@ def _get_children(self, max_count: int) -> None:
...
+                self._children.insert(
+                    0,
+                    (
+                        'LLDB ERROR',
+                        # In order to do this we need our error message as a valid C++
+                        # expression. We pack it into a raw string literal with a custom
+                        # delimiter to avoid needing to deal with escaping.
+                        # In order to do this we need our error message as a valid C++
+                        # expression. We pack it into a raw string literal with a custom
+                        # delimiter to avoid needing to deal with escaping.
```
Comment (slackito) [inline gdb/printing.py]: The comment is pasted twice.

**Review by slackito — APPROVED** [review]
Just one minor thing. Apart from that, LGTM.

It would be a good idea to add tests for this new behavior in a follow-up commit if you have the time.

**Pull request merged**